src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -628,16 +625,27 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return studentList;
     }
 
-    private List<String> getInvalidityInfoInDuplicatedEmail(String email,
-            List<String> studentEmailList, String[] linesArray) {
-        List<String> info = new ArrayList<>();
-        info.add("Same email address as the student in line \"" + linesArray[studentEmailList.indexOf(email) + 1] + "\"");
+    private String getInvalidityInfoInDuplicatedEmail(String email,
+            List<StudentAttributes> studentList, String[] linesArray) {
+        String info = null;
```

Comment:
`info` isn't necessary. Remove line 635 and have the return statement on line 634 instead.